shared/constants/transaction.js

```diff
@@ -11,7 +11,8 @@ import { MESSAGE_TYPE } from './app';
  * @property {'approve'} TOKEN_METHOD_APPROVE - A token transaction requesting an
  *  allowance of the token to spend on behalf of the user
  * @property {'incoming'} INCOMING - An incoming (deposit) transaction
- * @property {'sentEther'} SENT_ETHER - A transaction sending ether to a recipient
+ * @property {'sentEther'} SENT_ETHER - A transaction sending ether to a recipient (LEGACY - replaced by sendingNativeAsset)
```

Comment:
[Q] Does the sentEther type still exist?

Reply:
It would be a legacy transaction type that we could safely remove later on, in case say a user has a pending simple send transaction when the version update takes place. I kept it as a legacy type instead of creating a migration for this edge case.

Reply:
if a user has a sentEther transaction pending, it would continue to be a sentEther indefinitely right? What mechanism updates a transaction's type?

Reply:
I think a migration is required here to treat the sentEther type as legacy, or to ever remove it.

Reply:
Sorry I was mistaken. We would need a migration to update transaction types. I've implicitly made it legacy by swapping out `sentEther` as a type we set as a transaction type for `simpleSend`, but kept the logic to support transactions still typed as `sentEther` which would either need to be left there (because it is used for ui related logic with activity log) or migrate and swap all transaction type instances of `sentEther` for `simpleSend`. I was being lazy but a migration is probably called for here.

Reply:
Added a migration. Should I remove the 'sentEther' type from the `TRANSACTION_TYPES` and replace the reference to this (enum) object in old migrations as well? cc @brad-decker @danjm

Reply:
Kind of a conundrum. Replacing the value in old migrations obfuscates what the migration was actually doing. I might, instead, create a local variable for SENT_ETHER in the migration and note what it is and why it exists.